README.md

```diff
@@ -1,32 +1,14 @@
-**Table of Contents**
+# Riax ![Build](https://github.com/lambdaclass/elixir_riak_core/actions/workflows/github-actions.yml/badge.svg)
 
-1. [Riax](#riax)
-2. [Riak Core:](#riak-core)
-    - [What is it?](#what-is-it)
-    - [Why Riax?](#why-riax)
-    - [What's so great about it?](#whats-so-great-about-it)
-    - [Use cases:](#use-cases)
-    - [More about Hashing and VNodes:](#more-about-hashing-and-vnodes)
-3. [Setup:](#setup)
-    - [Single node:](#single-node)
-    - [Multiple nodes:](#multiple-nodes)
-4. [Tutorial:](#tutorial)
-    - [Use Case:](#use-case)
-    - [Solution:](#solution)
-    - [Creating project.](#creating-project)
-    - [Limiting VM Memory.](#limiting-vm-memory)
-    - [Setting up csv.](#setting-up-csv)
-    - [Storing:](#storing)
-    - [Reading CSV:](#reading-csv)
-    - [Visualizing Results:](#visualizing-results)
-
-# Riax
 Riax is an Elixir wrapper for Riak Core. 
-Riak Core is a distributed systems framework, written in Erlang.
-You can think of it as a building block for distributed and scalable systems.
+Riak Core is a building block for distributed and scalable systems
+in the form of an Erlang Framework.
+To learn more, you can check the useful links section in the repo linked above, and check the 
+[docs](https://lambdaclass.github.io/elixir_riak_core/readme.html) for more.
```

Comment:
instead of "the docs" you can say separately "the tutorial"  and "the API reference"

Reply:
Done!